timescale-container/test/ql-db-init/2-mtvecciano.etdevice-import.sql

```diff
@@ -0,0 +1,17 @@
+CREATE SCHEMA IF NOT EXISTS "mtvecciano";
```

Comment:
i am not sure i would refer to vecciano in a public repo.

Reply:
got rid of it, thx!